Automattic/pocket-casts-ios
# [Paywall Experiments] Features Carousel variant

| 📘 Part of: #1986
|:---:|

Fixes #2061

This PR introduces the Features Carousel Variant


https://github.com/user-attachments/assets/0bc007a5-a721-4cfc-8a39-332385ce2f17

## To test

> [!NOTE]
> Make sure to enable the `upgradeExperiment` feature flag

- CI must be 🟢 
- Use a non-plus account 
- Go to Profile -> Settings and tap on `Pocket Casts Plus`
- Check the new carousel like the video above

## Checklist
